integration/testrunner/config.go

```diff
@@ -60,6 +50,7 @@ type Config struct {
 	IDToken                 string        `env:"AUDIT_CLIENT_TEST_IDTOKEN"`
 	ProjectID               string        `env:"AUDIT_CLIENT_TEST_PROJECT_ID,required"`
 	BigQueryDataset         string        `env:"AUDIT_CLIENT_TEST_BIGQUERY_DATASET,required"`
+	PrivateKeyFilePath      string        `env:"PRIVATE_KEY_PATH,required"`
```

Comment:
Do we need to add "AUDIT_CLIENT_TEST" prefix for the env?

Reply:
As an alternative, will it be better just to hardcode the path in parsePrivateKey() instead of pass the path via config?